src/ports/postgres/modules/utilities/path.py_in

```diff
@@ -108,8 +107,11 @@ def path(schema_madlib, source_table, output_table, partition_expr,
                             *,
                             nextval('{seq_gen}') AS {id_col_name},
```

Comment:
What are you basing that on? I don't believe there's any significant performance benefit to sequences over row_number() (actually, a quick glance at the code leads me to think sequences will be slower). On top of that, creating a sequence adds to catalog bloat (not to mention the cost of creating and then dropping the sequence).